tests/pkcli/ci_data/check_prints1.in/x.py

```diff
@@ -0,0 +1,4 @@
+def check_prints():   
+    from pykern.pkdebug import pkdp
+    pkdp("this pkdp should be found")
```

Comment:
blank line above.

Reply:
Add the case of a pkdp and the module level also test for print. The function name "check_prints" doesn't really name what's going on. What would be a better name? The import statement can be at the global level, because this is a test script.